server/models/Group.js

```diff
@@ -0,0 +1,70 @@
+// @flow
+import { Op, DataTypes, sequelize } from '../sequelize';
+
+const Group = sequelize.define(
+  'group',
+  {
+    id: {
+      type: DataTypes.UUID,
+      defaultValue: DataTypes.UUIDV4,
+      primaryKey: true,
+    },
+    teamId: {
+      type: DataTypes.UUID,
+      defaultValue: DataTypes.UUIDV4,
+    },
+    name: {
+      type: DataTypes.STRING,
+      allowNull: false,
+    },
+  },
+  {
+    timestamps: true,
```

Comment:
```suggestion
    timestamps: true,
    paranoid: true,
```

Lets keep deleted groups in the db

Reply:
there's a naming uniqueness constraint, how does that play with paranoid?

Reply:
The uniqueness is based on a custom query so it should be a non-issue, deleted groups will be filtered by default